Stanford/Utility/Install/CreateMenuLinks.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @file
+ * This class creates new menu links.
+ */
+
+namespace Stanford\Utility\Install;
+
+class CreateMenuLinks extends \AbstractInstallTask {
+
+
+  public function execute(&$args = array()) {
+    /**
```

Comment:
I think we can use `7.x-1.x` for that. We're close. And nothing depends on `7.x-1.x` being stable.